protocol/xmrmaker/swap_state_test.go

```diff
@@ -183,10 +177,6 @@ func TestSwapState_GenerateAndSetKeys(t *testing.T) {
 }
 
 func TestSwapState_ClaimFunds(t *testing.T) {
-	if testing.Short() {
-		t.Skip() // TODO: randomly fails on CI with "no contract code at given address"
-	}
```

Comment:
I can put this back if it is still happening.  It sounds like the error you might get if you don't wait for the contract creation to get mined into a block.

Reply:
if it's not happening anymore, that's great :D